<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Стоп слова: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="stopWordLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="browseAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Сервис</string>
    </property>
    <addaction name="searchAction"/>
    <addaction name="separator"/>
    <addaction name="textAnalysisAction"/>
    <addaction name="graphicAnalysisAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="browseButton">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="browseAction">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="searchAction">
   <property name="text">
    <string>Поиск</string>
   </property>
  </action>
  <action name="textAnalysisAction">
   <property name="text">
    <string>Тесктовый анализ</string>
   </property>
  </action>
  <action name="graphicAnalysisAction">
   <property name="text">
    <string>Графический анализ</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
